src/autoscaler/mta.tpl.yaml

```
@@ -3,15 +3,17 @@ description: Application Autoscaler Release for Cloud Foundry
 _schema-version: "3.3.0"
 provider: Cloud Foundry Foundation
 copyright: Apache License 2.0
-version: VERSION
+version: MTA_VERSION
 
 modules:
   - name: metricsforwarder
     type: go
     path: .
     properties:
-      GO_INSTALL_PACKAGE_SPEC: code.cloudfoundry.org/app-autoscaler/src/autoscaler/metricsforwarder/cmd/metricsforwarder
       DT_RELEASE_BUILD_VERSION: ${mta-version}
+      GO_INSTALL_PACKAGE_SPEC: code.cloudfoundry.org/app-autoscaler/src/autoscaler/metricsforwarder/cmd/metricsforwarder
+      GOTOOLCHAIN: local
+      GOVERSION: go1.GO_MINOR_VERSION.x
```

Does this work this work setting the patch version to `x`? Why not leaving it out completely? For example 
```suggestion
      GOVERSION: go1.GO_MINOR_VERSION
```

---

Should work, yes.

---

What's the advantage? leaving out `.x` looks a bit simpler to me 🤓

---

Scratch that, didn't see that you simplified it already :D